airflow/api/common/mark_tasks.py

```diff
@@ -33,8 +33,8 @@
 if TYPE_CHECKING:
     from sqlalchemy.orm import Session as SASession
 
-    from airflow.models.dag import DAG
     from airflow.models.operator import Operator
+    from airflow.sdk.defiintion.dag import DAG
```

Comment:
```suggestion
    from airflow.sdk.defintion.dag import DAG
```

Reply:
Although, feel free to merge it if CI passes -- you can do this as a follow-up

Reply:
Oh, most of the properties this needs don't exist on SDK dag.